client/src/_shared/LoggedInLayout.js

```diff
@@ -14,6 +14,8 @@ const { useBreakpoint } = Grid
 export function LoggedInLayout({ children, title }) {
   const dispatch = useDispatch()
   const { t, i18n } = useTranslation()
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth)
```

Comment:
😭  Ant Design, why you make us do this

Reply:
lol I know...it makes sense what they are doing with their breakpoints, bc they're going with min-width, but I wish that screens object was a little different. sigh